stable/concourse/values.yaml

```diff
@@ -678,6 +678,16 @@ concourse:
       ##
       useCaCert: false
 
+    sidecarContainers:
+    - name: atc-syslog
+      image: splunk/scs
+      ## See https://splunk-connect-for-syslog.readthedocs.io/en/master/gettingstarted/docker-systemd-general/#configure-the-sc4s-environment
+      env:
+      - name: SPLUNK_HEC_URL
+        value: https://http-inputs-opendoor.splunkcloud.com:443
```

Comment:
URL scheme is `https`.  Is `:443` required?

Reply:
I don't know what format the `splunk/scs` container expects so added it to be safe. I can try dropping the `:443` when testing on `concourse-manual`.